### ISSUE TITLE
No space left on device - reduce logger level

### ISSUE THREAD

**Issue by bew** [crystal-lang-tools/scry#67]
I usually work with _large_ files (300-400 lines), and the logging file can become very big and take all the space available in `/tmp` (which is a tmpfs on my system, so it lives in the ram).

A sample list of scry's log files:
```
-rw-------  1 lesell_b users  3.1G Mar 21 20:25 scry.out.27K0Q6
-rw-------  1 lesell_b users  1.1M Mar 21 17:24 scry.out.DJW0Dc
-rw-------  1 lesell_b users 1014K Mar 21 17:22 scry.out.KqqV1g
-rw-------  1 lesell_b users  6.8M Mar 21 23:31 scry.out.lyxMHz
-rw-------  1 lesell_b users 1014K Mar 21 17:23 scry.out.M8PRUt
-rw-------  1 lesell_b users 1013K Mar 21 17:22 scry.out.TShzqU
-rw-------  1 lesell_b users 1014K Mar 21 17:21 scry.out.VkHbyS
-rw-------  1 lesell_b users  845M Mar 21 23:27 scry.out.XTtliD
-rw-------  1 lesell_b users 1020K Mar 21 17:21 scry.out.ZtJysi
```

I suggest to _at least_ disable the logging of every request/response unless the `--debug` flag is given to scry, this will reduce the size of log files significantly I think.

**Comment by faustinoaq** [comment]
Hi, @bew Scry already reduce the amount of logging by using `Log.logger.debug` when `log_level == "debug"`, Are you able to setup this in your editor?

By example in VSCode I use `crystal-lang.logLevel = "debug"` (1) when I want to get full logs otherwise normal logs should looks like:

```
-rw-------  1 main main 21K Mar 20 14:38 scry.out.fYKWU7
-rw-------  1 main main 22K Mar 20 15:43 scry.out.gJ2tFR
-rw-------  1 main main 37K Mar 20 16:24 scry.out.PoePdH
-rw-------  1 main main 27K Mar 20 17:39 scry.out.J4kKR0
-rw-------  1 main main 18K Mar 20 19:56 scry.out.4E4QX0
-rw-------  1 main main 30K Mar 20 21:45 scry.out.5ek97p
-rw-------  1 main main 15K Mar 20 22:21 scry.out.XW0lyb
-rw-------  1 main main 14K Mar 20 23:32 scry.out.9Qqf80
-rw-------  1 main main 47K Mar 21 10:34 scry.out.jPzZ7E
-rw-------  1 main main 27K Mar 21 10:41 scry.out.FWe8aj
-rw-------  1 main main 12K Mar 21 10:55 scry.out.VAHVqd
-rw-------  1 main main 37K Mar 21 11:48 scry.out.n5uPyV
-rw-------  1 main main 19K Mar 21 14:59 scry.out.yEgFju
-rw-------  1 main main 44K Mar 21 15:41 scry.out.YQAwY6
-rw-------  1 main main 33K Mar 21 17:21 scry.out.VkHbyS
-rw-------  1 main main 12K Mar 21 17:22 scry.out.KqqV1g
-rw-------  1 main main 31K Mar 21 17:22 scry.out.TShzqU
-rw-------  1 main main 36K Mar 21 17:23 scry.out.M8PRUt
-rw-------  1 main main 30K Mar 21 17:24 scry.out.DJW0Dc
-rw-------  1 main main 23K Mar 21 18:24 scry.out.5rdbp5
-rw-------  1 main main 17K Mar 21 18:48 scry.out.30AZNG
-rw-------  1 main main 39K Mar 21 20:25 scry.out.27K0Q6
-rw-------  1 main main 24K Mar 21 21:56 scry.out.jeIOjF
-rw-------  1 main main 37K Mar 21 23:27 scry.out.8oMpvl
-rw-------  1 main main 15K Mar 21 23:31 scry.out.lyxMHz
-rw-------  1 main main 32K Mar 21 23:36 scry.out.DZGDpG
```

1. https://github.com/crystal-lang-tools/vscode-crystal-lang/wiki/Settings#loglevel

**Comment by laginha87** [comment]
The log gets updated by the editor here  https://github.com/crystal-lang-tools/scry/blob/master/src/scry/update_config.cr when scry receives DidChangeConfigurationParams after initialization.

**Comment by keplersj** [comment]
Dropping the default log level down to `info` - or even `warning` - seems like good idea to me.

**Comment by faustinoaq** [comment]
> Dropping the default log level down to info - or even warning - seems like good idea to me.

Yes, I like the idea :+1:

**Comment by faustinoaq** [comment]
Fixed by #93 

`logger.debug` should be used only during scry development, for final user the log files can grow pretty huge like @bew said:

![screenshot_20180421_195116](https://user-images.githubusercontent.com/3067335/39090176-5b6e8c82-459d-11e8-8bee-c1856c70753c.png)
